Ginger/GingerCoreNET/Run/RunsetExecutor.cs

```diff
@@ -336,6 +355,10 @@ public void RunRunset(bool doContinueRun = false)
                 //reset run       
                 if (doContinueRun == false)
                 {
+                    if (WorkSpace.Instance.RunningInExecutionMode == false || RunSetConfig.ExecutionID == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5538205)